cmd/helmvm/install.go

```diff
@@ -310,6 +313,12 @@ var installCommand = &cli.Command{
 		},
 	},
 	Action: func(c *cli.Context) error {
+		if defaults.DecentralizedInstall() {
+			logrus.Warnf("Decentralized install was detected. To manage the cluster")
+			logrus.Warnf("you have to use the '%s node' commands instead.", defaults.BinaryName())
+			logrus.Warnf("Run '%s node --help' for more information.", defaults.BinaryName())
+			return nil
```

Comment:
seems like you should return an error here

Reply:
Done.